Lib/KeychainAccess/Keychain.swift

```diff
@@ -884,7 +884,15 @@ public final class Keychain {
             throw securityError(status: status)
         }
     }
-
+    
+    public func contains(_ key: String, withoutAuthenticationUI: Bool = false) throws -> Bool {
+        return try self.contains(query:[AttributeAccount:key])
```

Comment:
@nd-net you haven't passed `withoutAuthenticationUI` into base function

Reply:
My business account portfolio